flytekit/bin/entrypoint.py

```diff
@@ -219,7 +218,7 @@ def setup_execution(
                 "api_version": _api_version,
             },
         ),
-        logging=_logging,
+        logging=logger,
```

Comment:
just a curiosity: it looks like this is not used?

Reply:
actually i'm going to revert this.  this is the logging handler passed to the user.